server/app/controllers/TransactionsController.scala

```diff
@@ -61,7 +61,7 @@ class TransactionsController @Inject()(transactionRPCService: TransactionRPCServ
           if (cacheable) {
             response.withHeaders("Cache-Control" -> "public, max-age=31536000")
           } else {
-            response.withHeaders("Cache-Control" -> "no-store")
+            response.withHeaders("fromOutputScriptASMCache-Control" -> "no-store")
```

Comment:
Header corrupted, you can catch these by reviewing your own PRs, which is commonly a good practice.